tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/deid/survey_version_info_test.py

```diff
@@ -40,7 +41,6 @@ def setUpClass(cls):
         # setting the cope lookup to the same dataset as the qrid map for testing purposes
```

Comment:
Luis has been collapsing lines 31 and 32 like so:
`cls.project_id = os.environ.get(PROJECT_ID)`

which would be a nice little cleanup here.